### ISSUE TITLE
ksp 1.4.x update

### ISSUE THREAD

**Issue by raidernick** [KSP-RO/RealismOverhaul#1880]
@PhineasFreak Just wondering if we want to do another 1.3.1 release or two before looking at 1.4.x or later versions of KSP. RO itself as is looks for the most part good to go for 1.4.x at any time. All of the dependencies on the RO github except RSS can be compiled and work in 1.4.x also. RSS will need some work as apparently it's broken with kopernicus, if that's something you want to look into. The only real thing holding up a release would be FAR, I don't know where @ferram4 is on a release for that.

So do we want to do another 1.3.1 release or two, go right to 1.4.x or wait for the next major ksp version(which I heard rumors may be coming soon)? Unless FAR is updated we will have to stick to 1.3.1 releases for now anyway though.

**Comment by PhineasFreak** [comment]
I have problems getting RealFuels to load correctly (tanks are fine but the engine modules throw nullrefs) so i cannot confirm if anything is broken on the RO side. All other plugins work and from some limited testing i can confirm that we could easily release the next RO version for KSP 1.4.X.

RSS works perfectly out of the box. I downloaded the current (KSP 1.3) releases, did a recompile for the plugin and created a quick KSP 1.4.3 dev installation. Everything checked out OK (i have no idea what the guy in the KSP forums is talking about, AFG works as expected).

FAR is usually the dependency that RO awaits for before doing a release for a new KSP version. Haven't seen @ferram4 around here for a while though.

**Comment by raidernick** [comment]
So depending on what the deal with FAR is we can expect to release at least 1 more RO, I plan to do it once a month from now on.

Good to know RSS works and that guy was full of it, getting RF fixed won't be an issue @blowfishpro and @Starwaster are always working on it.

Also, while RO may be 1.4 ready, it needs the new DLC parts added to it. I don't have the dlc and have no intentions of supporting TT and paying the insane amount of money they are asking to get it either. Someone who has it will need to do the configs.

**Comment by PhineasFreak** [comment]
I don't have the DLC either and, judging from the content/price ratio, i do not think that i will ever get it. I also think that they are separate parts and do not replace any of the "stock" content (it does add some extra coloring variants to them though).

**Comment by raidernick** [comment]
I saw posts on the forum that it "disables" some of the stock parts.

**Comment by blowfishpro** [comment]
It doesn't disable anything.  That's probably referring to the old Mk1-2 Command pod and Size 2 tanks which are hidden to avoid breaking craft files.  That's with or without the DLC though.

**Comment by rsparkyc** [comment]
Even if we're doing another 1.3.1 release, is there any objection to begin cutting 1.4.3 releases of the sub components?  For example, I just recompiled RealHeat, want me to make a release for that?

**Comment by raidernick** [comment]
RO is waiting on FAR and I'm doing monthly releases of RO now, so if the new FAR isn't out by then another 1.3.1 release will happen. As for the dependencies, all of them except RSS can be updated sure.

**Comment by Bornholio** [comment]
1.4.3 sooner the better in my mind.  most of the important mods are somewhere 1.4.1-1.4.3 range
Should open an issue to add the DLC as supported mod

**Comment by PhineasFreak** [comment]
With KSP 1.4.4 more or less confirmed i think that we should wait until then.

**Comment by rsparkyc** [comment]
Now that 1.4.5 is out, and there should be no 1.4.6, it's probably time we re-address this.  Obviously a big blocker is FAR, though there are PRs out for that.  @raidernick, do you want to add a checklist to the head of this issue, or should we start a new one?  Do we want to use the newest Golden Spreadsheet to track stuff?  I'm up for anything.

**Comment by raidernick** [comment]
we can use the checklist, make a new issue and put the link at the top. I don't even plan to go through the RO pr's until FAR has an actual real official release, then I'll make sure this is put out. But this still does need someone with the stupid microtransaction content to make patches for those parts/changes.

**Comment by Bornholio** [comment]
Rsparkyc thanks for the drive on required mods.  https://docs.google.com/spreadsheets/d/1jqFjaHCo0gdIj5mX61V8D02Ma6v1WfH2Zuv0ddXgG5M/edit#gid=0 I've been updating weekly or better lately but  only checking via CKAN.  Yeah FAR is a big blocker.  Also KCT and such has been abandoned and offered up for new stweardship.

**Comment by PhineasFreak** [comment]
Maybe fork FAR into KSP-RO for our purposes temporarily? Dunno, our options are very limited (wait for **Ferram** to return or for someone to create an official fork).

**Comment by jwvanderbeck** [comment]
Some of you should have sufficient permissions to do whatever you need to do in the TF repo, so feel free to do so.
